lookout/style/typos_checker/analyzer.py

```diff
@@ -34,20 +34,20 @@ def analyze(self, ptr_from: ReferencePointer, ptr_to: ReferencePointer,
                 for index, comment_node in enumerate(comment_nodes):
                     if index in suggestions.keys():
                         corrections = suggestions[index]
+                        line_suggestions = ""
                         for token in corrections.keys():
-                            comment = Comment()
-                            comment.file = change.head.path
                             corrections_line = ""
                             for candidate in corrections[token]:
-                                corrections_line += candidate[0] + " (%d)," % \
-                                                    int(candidate[1] * 100)
-                            comment.text = "Typo inside comment in token '%s'. " \
-                                           "Possible corrections:" % token +\
-                                           corrections_line[:-1]
-                            comment.line = comment_node.start_position.line
-                            comment.confidence = int(
-                                corrections[token][0][1] * 100)
-                            comments.append(comment)
+                                corrections_line += " `%s` (%d)," % \
+                                                    (candidate[0], int(candidate[1] * 100) )
+                            line_suggestions += "Typo inside comment in word `%s`. " \
```

Comment:
Agree with @meyskens on the message structure. I think we should distinguish different typos more clearly. So either make some kind of a list, or markdowns (that won't work locally though), or just '\n'.

Reply:
Agree with you @irinakhismatullina and @meyskens; I'll add an extra `\n` between different typos.
(we can work in a better layout in a further iteration)